odoo/addons/base/models/res_lang.py

```diff
@@ -351,6 +370,16 @@ def action_activate_langs(self):
             }
         }
 
+    def name_get(self):
```

Comment:
Shouldn't it be limited to some usage using a context key (like the m2o on lead), and keep the basic behavior everywhere else ? Especially in stable ?